pkg/steps/lease_test.go

```diff
@@ -29,10 +29,12 @@ func (s *stepNeedsLease) Run(ctx context.Context, dry bool) error {
 	return nil
 }
 
-func (stepNeedsLease) Name() string             { return "needs_lease" }
-func (stepNeedsLease) Description() string      { return "this step needs a lease" }
-func (stepNeedsLease) Requires() []api.StepLink { return []api.StepLink{api.ReleaseImagesLink()} }
-func (stepNeedsLease) Creates() []api.StepLink  { return []api.StepLink{api.ImagesReadyLink()} }
+func (stepNeedsLease) Name() string        { return "needs_lease" }
+func (stepNeedsLease) Description() string { return "this step needs a lease" }
+func (stepNeedsLease) Requires() []api.StepLink {
+	return []api.StepLink{api.StableImagesLink(api.LatestStableName)}
```

Comment:
As there's no real generic "release images" intermediary, all of these have changed to explicitly depend on the full presence of the "stable" image stream. This should be a no-op.